src/js/components/Skeleton/stories/Group.js

```diff
@@ -0,0 +1,68 @@
+import React, { useContext } from 'react';
+
+import { Box, Button, Card, Grommet, Image, Text, ThemeContext} from 'grommet';
+import { FormAdd } from 'grommet-icons';
+
+const Item = ({ ...rest }) => (
+  <Card
+    width="531px"
+    pad="medium"
+    gap="large"
+    round="medium"
+    flex="grow"
+    border
+    {...rest}
+  >
+    <Box direction="row" gap="large" justify="between" align="center">
+      <Box direction="row" gap="medium">
+        <Box width="96px" height="96px" background="orange!" round="small">
```

Comment:
can we just use `Avatar` size `xlarge`?

Reply:
This is more of just a product image and not really an avatar. Like with Marketplace cards where we have the product icon which is an image. Avatar is intended to be more of a circle.

Reply:
okay thats fair